app/views/verify_your_email.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
```

Comment:
Better to move this file into views.email folder as there would be a page for undeliverable email